.github/workflows/checks.yml

```
@@ -0,0 +1,31 @@
+name: Checks
+on:
+  pull_request:
+    branches:
+      - main
+jobs:
+  size:
+    runs-on: ubuntu-latest
+    name: Checking size
+    if: "!contains(github.event.head_commit.message, '[ci skip]')"
+    env:
+      CI_JOB_NUMBER: 1
+    steps:
+      - name: Checkout the repository
+        uses: actions/checkout@v2
+      - name: Install pnpm
+        uses: pnpm/action-setup@v2.0.1
+        with:
+          version: 6
+      - name: Install Node.js
+        uses: actions/setup-node@v2
+        with:
+          node-version: 16
+          cache: 'pnpm'
+      - name: Install dependencies
+        run: pnpm install
+      - name: Check size
+        uses: andresz1/size-limit-action@v1
+        with:
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+          skip_step: install
```

```suggestion
      - name: Check size
        uses: andresz1/size-limit-action@master
        with:
          github_token: ${{ secrets.GITHUB_TOKEN }}
```

code from master has pnpm support

---

changed

---

@thabarbados look at the suggestion carefully

---

@dangreen sorry, fixed